css/desktop.css

```diff
@@ -73,25 +71,32 @@ header {
 
 .section2-container {
   .wrapper {
-    #article-2 {
-      position: relative;
 
-      h3 {
-        font-size: 14pt;
-        font-weight: normal;
-        text-align: center;
-      }
+    h3 {
+      display: flex;
+      width: 80%;
+      margin: 0 auto;
 
-      .inner-article {
-        img {
-          margin-top: 80px;
-          margin-right: 50px;
-          width: 460px;
-          float: left;
-        }
+      font-size: 15pt;
```

Comment:
Why you mixed 'pt, px' ?

Reply:
I thought using pt for font-size seemed more specific, am I wrong?

Reply:
There is no right answer but I prefer uniformity